relevanceai/operations/vector/vectorize.py

```diff
@@ -135,16 +155,6 @@ def vectorize(
 
         fields = image_fields + text_fields
 
-        foreign_fields = []
```

Comment:
I think we should always check fields. This is a really good check to see if a field is actually in the dataset